sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc

```diff
@@ -8,7 +8,7 @@ The `sycl::property_list` found in SYCL 2020 is used to store properties used in
 
```

Comment:
Commenting here because I can't add comments on the earlier lines ...

I think we should rename this extension from `SYCL_EXT_ONEAPI_PROPERTY_LIST` to`SYCL_EXT_ONEAPI_PROPERTIES`.  The shorter name is nice and the main class in this extension is now named `properties` instead of `property_list`.  If we do this, the name of the file should also be renamed to `SYCL_EXT_ONEAPI_properties.asciidoc`.

Reply:
I'm OK either way. What does everyone else think?

Reply:
We cannot use the same name as both a feature-test macro and also something else.  Therefore, if this extension has the feature-test macro `SYCL_EXT_ONEAPI_PROPERTIES`, we would be precluded from using that same name as a way to specify properties for a free function.

I had forgotten that we defined `SYCL_EXT_ONEAPI_PROPERTY` as a way to specify properties on a free function.  It does seem possibly confusing that the name is so similar to the feature-test macro `SYCL_EXT_ONEAPI_PROPERTIES` (or `SYCL_EXT_ONEAPI_PROPERTY_LIST`).  The confusion seems similar either way, though, so I think I still prefer `SYCL_EXT_ONEAPI_PROPERTIES`.

Maybe we should rename the macro for specifying properties on free functions to `SYCL_EXT_ONEAPI_FUNCTION_PROPERTY`?

Reply:
> Maybe we should rename the macro for specifying properties on free functions to `SYCL_EXT_ONEAPI_FUNCTION_PROPERTY`?

That sounds like a good idea to me.  I'll make that change in the KernelProperties spec.